UPDATING.md

```diff
@@ -84,6 +84,20 @@ constantly.
 From Airflow 2.0.0, the scheduling decisions have been moved from
 DagFileProcessor to Scheduler, so we can keep the default a bit higher: `30`.
 
+### SFTP connection `private_key_pass` extra param is renamed to `private_key_passphrase`
+
+Some operators perform SFTP operations by the means of `SFTPHook`. `SFTPHook` delegates the actual SFTP connection to
+ the `pysftp` module.
+
+When connecting with a private key protected by a passphrase, the `private_key_pass` extra parameter had to be
+ specified in the connection, and this corresponds to the arguments' naming in `pysftp`.
+
+However, `SFTPHook` inherits from `SSHHook` which already manages a `private_key_passphrase` extra param. But this
+ param was unused in favor of the `private_key_pass` param introduced in `SFTPHook`.
+
+For consistency, `private_key_pass` has been dropped in `SFTPHook` and `private_key_passphrase` is well reused.
+This way, a single connection can be used for both SSH and SFTP operations.
+
```

Comment:
I don't think this is the place for this.
It's information about the backward incompatibility of a provider. 
If I upgrade a provider I need to see this in the provider release notes not in Airflow release notes.

Reply:
Correct. Those should be moved to CHANGELOG.rst (and ADDITIONAL_INFO.md for backports till the end of March). Those are newly added (just merged some changes recently and once this PR is merged (#14013) the SFTP provider will have both files that should be  modified instead. I am releasing new wave of providers today. so this change wil only male it to the next one (but I think it is not urgent and can be released easily in 2-3 weeks when we release the next wave).

Reply:
:heavy_check_mark: Done

Reply:
Hi,
I don't know if it's my job to resolve the conversation !?